tests/ui/conftest.py

```diff
@@ -32,35 +42,100 @@ def jwt_issuer(base_url, variables):
 @pytest.fixture(scope='session')
 def jwt_secret(base_url, variables):
     try:
-        hostname = [urlparse.urlsplit(base_url).hostname]
+        hostname = urlparse.urlsplit(base_url).hostname
         return variables['api'][hostname]['jwt_secret']
     except KeyError:
         return os.getenv('JWT_SECRET')
 
 
 @pytest.fixture
-def jwt_token(base_url, jwt_issuer, jwt_secret):
-    payload = {
-        'iss': jwt_issuer,
-        'iat': datetime.datetime.utcnow(),
-        'exp': datetime.datetime.utcnow() + datetime.timedelta(seconds=30)}
-    return jwt.encode(payload, jwt_secret, algorithm='HS256')
+def initial_data(transactional_db):
+    from olympia.amo.tests import addon_factory, user_factory
+    from olympia.addons.models import AddonUser
+
+    for x in range(10):
+        AddonUser.objects.create(user=user_factory(), addon=addon_factory())
+
+
+@pytest.fixture
+def create_superuser(transactional_db, my_base_url, tmpdir, variables):
+    create_switch('super-create-accounts')
+    call_command('loaddata', 'initial.json')
+
+    call_command(
+        'createsuperuser',
+        interactive=False,
+        username='uitest',
+        email='uitester@mozilla.org',
+        add_to_supercreate_group=True,
+        save_api_credentials=str(tmpdir.join('variables.json')),
+        hostname=urlparse.urlsplit(my_base_url).hostname
+    )
+
+    with tmpdir.join('variables.json').open() as f:
+        variables.update(json.load(f))
 
 
 @pytest.fixture
-def user(base_url, fxa_account, jwt_token):
-    user = {
+def user(create_superuser, my_base_url, fxa_account, jwt_token):
+    url = '{base_url}/api/v3/accounts/super-create/'.format(
+        base_url=my_base_url)
+
+    params = {
         'email': fxa_account.email,
         'password': fxa_account.password,
-        'username': fxa_account.email.split('@')[0]}
-    url = '{base_url}/api/v3/accounts/super-create/'.format(base_url=base_url)
-    params = {
-        'email': user['email'],
-        'username': user['username'],
-        'password': user['password'],
+        'username': fxa_account.email.split('@')[0],
         'fxa_id': fxa_account.session.uid}
     headers = {'Authorization': 'JWT {token}'.format(token=jwt_token)}
-    r = requests.post(url, data=params, headers=headers)
-    assert requests.codes.created == r.status_code
-    user.update(r.json())
-    return user
+    response = requests.post(url, data=params, headers=headers)
+    assert requests.codes.created == response.status_code
+    params.update(response.json())
+    return params
+
+
+@pytest.fixture(scope='function')
+def live_server(request, transactional_db):
+    """
+        This fixture overrides the live_server fixture provided by
+        pytest_django. live_server allows us to create a running version of the
+        addons django application within pytest for testing.
+
+        Christopher Grebs:
```

Comment:
Let's change that maybe to `cgrebs` which is what I'm called everywhere (IRC, mail etc)